docs/faq.md

```diff
@@ -19,9 +19,6 @@ Please read this page before asking a question in the Discord. Your help request
 
 **Can I get AVS or SOF drivers for free?**
 * Only the Linux drivers are free.
- 
```

Comment:
Instead of deleting, mention that using the latest release instead of lts will work better.